azuread/table_azuread_user.go

```diff
@@ -43,72 +46,70 @@ func tableAzureAdUser() *plugin.Table {
+			{Name: "surname", Type: proto.ColumnType_STRING, Description: "Family name or last name of the active directory user.", Transform: transform.FromMethod("GetSurname")},
 
 			{Name: "filter", Type: proto.ColumnType_STRING, Transform: transform.FromQual("filter"), Description: "Odata query to search for resources."},
 
 			// Other fields
-			{Name: "on_premises_immutable_id", Type: proto.ColumnType_STRING, Description: "Used to associate an on-premises Active Directory user account with their Azure AD user object."},
-			{Name: "created_date_time", Type: proto.ColumnType_TIMESTAMP, Description: "The time at which the user was created."},
-			{Name: "mail", Type: proto.ColumnType_STRING, Description: "The SMTP address for the user, for example, jeff@contoso.onmicrosoft.com."},
-			{Name: "mail_nickname", Type: proto.ColumnType_STRING, Description: "The mail alias for the user."},
-			{Name: "password_policies", Type: proto.ColumnType_STRING, Description: "Specifies password policies for the user. This value is an enumeration with one possible value being DisableStrongPassword, which allows weaker passwords than the default policy to be specified. DisablePasswordExpiration can also be specified. The two may be specified together; for example: DisablePasswordExpiration, DisableStrongPassword."},
+			{Name: "on_premises_immutable_id", Type: proto.ColumnType_STRING, Description: "Used to associate an on-premises Active Directory user account with their Azure AD user object.", Transform: transform.FromMethod("GetOnPremisesImmutableId")},
+			{Name: "created_date_time", Type: proto.ColumnType_TIMESTAMP, Description: "The time at which the user was created.", Transform: transform.FromMethod("GetCreatedDateTime")},
+			{Name: "mail", Type: proto.ColumnType_STRING, Description: "The SMTP address for the user, for example, jeff@contoso.onmicrosoft.com.", Transform: transform.FromMethod("GetMail")},
+			{Name: "mail_nickname", Type: proto.ColumnType_STRING, Description: "The mail alias for the user.", Transform: transform.FromMethod("GetMailNickname")},
+			{Name: "password_policies", Type: proto.ColumnType_STRING, Description: "Specifies password policies for the user. This value is an enumeration with one possible value being DisableStrongPassword, which allows weaker passwords than the default policy to be specified. DisablePasswordExpiration can also be specified. The two may be specified together; for example: DisablePasswordExpiration, DisableStrongPassword.", Transform: transform.FromMethod("GetPasswordPolicies")},
 			{Name: "refresh_tokens_valid_from_date_time", Type: proto.ColumnType_TIMESTAMP, Description: "Any refresh tokens or sessions tokens (session cookies) issued before this time are invalid, and applications will get an error when using an invalid refresh or sessions token to acquire a delegated access token (to access APIs such as Microsoft Graph)."},
```

Comment:
There's an issue open for this in https://github.com/microsoftgraph/msgraph-sdk-go/issues/227, are we getting data for this column, or have some workaround for it already?

Reply:
@cbruno10 As per the SDK team, they had logged an issue for adding those data in response.
https://github.com/microsoftgraph/msgraph-sdk-go/issues/227#issuecomment-1205188942

Currently, we have a workaround to get these data. I had already updated the table to gather those data
https://github.com/turbot/steampipe-plugin-azuread/pull/62/files#diff-0355132864b3978d9dee70e144ba0b6b1328b6c14a4afbb7f33bc610eaa88f70R143

Reply:
@cbruno10 I have removed column `refresh_tokens_valid_from_date_time`, since`refreshTokensValidFromDateTime` should not be present anymore. Use `sign_in_sessions_valid_from_date_time` instead.